City-of-Pori/pori-kada
# PORI-328 Added images to person card display

To test:

1. drush fra -y && drush cc all
2. Add image to any person
3. Go to https://local.pori.fi/phone-book and make sure that person has image visible and it looks good.
